### PR TITLE
Make plugin discovery Qt4 compatible

### PR DESCRIPTION
**Pull request by Kissaki**
Should fix #19

### CODE REVIEW

File: Plugins.cpp [hunk 1]
```diff
@@ -30,6 +30,11 @@
 */
 
 #include "Plugins.h"
+#if QT_VERSION >= 0x050400
+#include <QStandardPaths>
+#elsif
```
Comment (davidebeatrici) [inline Plugins.cpp]: #elif

File: Plugins.cpp [hunk 2]
```diff
@@ -73,7 +78,11 @@ Plugins::Plugins(QObject *p) : QObject(p) {
 	}
 #endif
 	// User plugins directory
+#if QT_VERSION >= 0x050400
 	QString appDataLocation = QStandardPaths::writableLocation(QStandardPaths::AppDataLocation);
+#elsif
```
Comment (davidebeatrici) [inline Plugins.cpp]: #elif

**Comment by Kissaki** [comment]
fixed

File: Plugins.cpp [hunk 3]
```diff
@@ -30,6 +30,11 @@
 */
 
 #include "Plugins.h"
+#if QT_VERSION >= 0x050400
+#include <QStandardPaths>
+#elif
```
Comment (davidebeatrici) [inline Plugins.cpp]: #else

File: Plugins.cpp [hunk 4]
```diff
@@ -73,7 +78,11 @@ Plugins::Plugins(QObject *p) : QObject(p) {
 	}
 #endif
 	// User plugins directory
+#if QT_VERSION >= 0x050400
 	QString appDataLocation = QStandardPaths::writableLocation(QStandardPaths::AppDataLocation);
+#elif
```
Comment (davidebeatrici) [inline Plugins.cpp]: #else

**Comment by davidebeatrici** [comment]
It should be _#else_, not _#elif_. My fault.

**Comment by Kissaki** [comment]
Fixed

**Comment by Kissaki** [comment]
Closing this as commit has been cherry-picked into #19 …

**Pull request closed without merging**